ISISNeutronMuon/MDANSE
# 3d view tab update

**Description of work**
Combined the trajectory and the 3d view tabs in one triple panel tab.

![Animation6](https://github.com/ISISNeutronMuon/MDANSE/assets/14276033/8d80f1ce-1bdc-4082-b6d6-cd662a94af65)


**Fixes**
Fixes #307, Fixes #303
Fixed so that the atom properties table updates correctly on trajectory change.

**To test**
1. Load a trajectory and select the trajectory. Check that the atom properties and information box updates correctly.
2. Load another trajectory and select it. Check that everything updates correctly.
3. Check that the play animations works.
4. Check the actions tab and the trajectories are there.
5. Run a analysis job and check that it worked properly.
6. Delete the trajectories in the trajectories tab and check everything updates correctly.
